src/bin/travis/testCentaurJes.sh

```diff
@@ -55,8 +55,8 @@ export PYTHONPATH="/usr/lib/python2.7/site-packages:/usr/local/lib/python2.7/sit
 export CONFIGURE_OPTS="--enable-unicode=ucs4"
 pyenv install 2.7.10
 pyenv global 2.7.10
-sudo pip install --upgrade pip
-sudo pip install pyopenssl ndg-httpsclient pyasn1 --upgrade
+sudo -H pip install --upgrade pip
+sudo -H pip install pyopenssl ndg-httpsclient pyasn1 --upgrade
```

Comment:
In the Travis logs there was a warning flashing by to add a "-H" flag for the pip commands. Left a link the comments for what the flag actually does.